src/Features/CSharp/Portable/CodeFixes/GenerateMethod/GenerateMethodCodeFixProvider.cs

```diff
@@ -50,9 +50,10 @@ internal static class GenerateMethodDiagnosticIds
 
     [ExportCodeFixProvider(LanguageNames.CSharp, Name = PredefinedCodeFixProviderNames.GenerateMethod), Shared]
     [ExtensionOrder(After = PredefinedCodeFixProviderNames.GenerateEnumMember, Before = PredefinedCodeFixProviderNames.PopulateSwitch)]
+    [ExtensionOrder(Before = PredefinedCodeFixProviderNames.AddParameter)]
```

Comment:
This should work in conjunction with the `[ExtensionOder]` of  the `CSharpAddParameterCodeFixProvider` but it didn't worked as expected. See my comment  above for details